docs/Deployment.md

```diff
@@ -10,51 +10,30 @@ BTCPay is a non-custodial invoicing system which eliminates the involvement of a
 Developer deployments are not recommended for production environments and require the user to have technical knowledge related to the build.
 :::
 
-![Decision diagram](./img/DecisionDiagInstallBTCPayServer.png "BTCPay Server Infographic")
 
-## What are my options?
 
-* [LunaNode Web Deployment](./LunaNodeWebDeployment.md) *
-* [Azure Deployment](./AzureDeployment.md) *
-* [Docker Deployment](https://github.com/btcpayserver/btcpayserver-docker/blob/master/README.md)
-* [Google Cloud Deployment](./GoogleCloudDeployment.md) *
-* [Hardware Deployment](./HardwareDeployment.md) *
-* [Third-Party Hosting](./ThirdPartyHosting.md)
-* [Manual Deployment](./ManualDeployment.md) **Not intended for production use**
-
-(*): Those deployments are using the Docker Deployment under the hood.
-
-## To chose one that will best suit your needs, consider the following:
-
-<br>
-
-|Web Solutions                             |1.                            |2.          |Why?          |
-| ---------------------------------------- |:----------------------------:|:----------:|:------------:|
-|Business<br>(Fast Setup)                  |[3rd Party <br> BTCPay Hosts](./ThirdPartyHosting.md)  |[LunaNode <br> Web-Wizard](./LunaNodeWebDeployment.md)* |- Low Difficulty<br>- BTCPayServer Support (1)<br>- Lightning Network (2)     |
-|Cost / Month                                     | Free                    | $10 |BTC Accepted  |
 
-**LunaNode Web-Wizard is a VPS solution, deployable from an easy-web interface.**
+## To chose one that will best suit your needs, consider the following:<br>
 
-<br>
+|Choosing a deployment <br>method      |   Difficulty   |    Production/Dev |      All features <br>available     |             Approx. cost<br> per month      |
+| -------------------------------- |:--------------:|:----------------------:|:---------------:|:-------------:|:-------------:|
```

Comment:
```suggestion
| -------------------------------- |:--------------:|:----------------------:|:---------------:|:-------------:|
```

The github preview of this table doesn't look right. Are you sure you need 6 segments here with only 5 columns?

Reply:
Oh I must have forgotten to delete that. I had a 6th column before making this PR.